components/layout/navbar/index.tsx

```diff
@@ -111,6 +114,9 @@ export default function Navbar({
                 <Cart />
               </Suspense>
             ) : null}
+            {profileMenu?.length ? (
+              <LoginButton domain={domain} profileMenu={profileMenu} />
```

Comment:
pass uiContent instead of profileMenu, it's always better to move object than simple string. you never know when it will be useful to have uiContent as params

Reply:
oui, je croyais que ce serait mieux de passer uniquement le necessaire mai sil est vrai que ce serait plus pratique davoir acces a tout?